### ISSUE TITLE
`System.loadLibrary` failed on MacOS when jvm created with invocation API

### ISSUE THREAD

**Issue by SergeevPavel** [jni-rs/jni-rs#220]
Steps to reproduce:
1. Create jvm with `JavaVM::new(args)`
2. Run some java code that will load native library.
3. If there is no jdk in your system whose plist have:
```
      <key>JVMCapabilities</key>
      <array>
        <string>CommandLine</string>
        <string>JNI</string>
      </array>
```
you will see 
![image](https://user-images.githubusercontent.com/4352041/72357356-e1ba5e00-36fb-11ea-84ae-15745b980388.png)
and program will terminated.

Proposed solution:
According to https://bugs.openjdk.java.net/browse/JDK-8064542 `JNI_CreateJavaVM` should be called from libjli.dylib. My experiments also show that it works fine.

**Comment by vitvakatu** [comment]
@dmitry-timofeev FYI https://jira.bf.local/browse/ECR-1930

**Comment by dmitry-timofeev** [comment]
Hi @SergeevPavel , thanks for reporting and investigating the issue! The fix must be in #230.

**Comment by dmitry-timofeev** [comment]
Released in https://crates.io/crates/jni/0.16.0
